### ISSUE TITLE
Suggest comparison to maven archetypes and JavaScript Yeoman

### ISSUE THREAD

**Issue by davidmichaelkarr** [pledbrook/lazybones#144]
I just heard about this from a presentation.  I briefly scanned the documentation, so I know the general idea.

I would suggest that you mention the similarity in the basic idea to Maven archetypes and the JavaScript Yeoman utility.  I'd also like to know if Lazybones has the additional feature that Yeoman has over Maven archetypes, which is the ability to integrate smaller pieces, instead of the entire application skeleton.  Yeoman can be used after the application has been created, just to integrate particular additional pieces.  Maven archetypes are just used to create the application skeleton. I noticed the concept of "sub-templates", which might be what this refers to, but I can't tell from a brief review. If this were to be possible, I assume that Lazybones would have to know to augment an existing build script, instead of creating a new one.  Does Lazybones do this kind of thing?

**Comment by pledbrook** [comment]
Yes, sub-templates are based on the same idea as Yeoman generators. Hopefully the [template developer's guide](https://github.com/pledbrook/lazybones/wiki/Template-developers-guide#user-content-sub-templates-since-07) will give you a good idea of what they are for. They only work inside a project that was created from a Lazybones template.

I'm not that familiar with Yeoman, so I hesitate to draw comparisons to it. Perhaps you could draw up a suitable mention in the project overview?

**Comment by davidmichaelkarr** [comment]
Actually, I've never used Yeoman either.  I understand the basic idea, and I immediately saw the comparison with Lazybones and Maven archetypes.  However, let me see if I can compose a simple statement covering this.
